vickev/howdypix
# Make the links SPA.

## Description

Make the links load the page in the browser rather than reloading the entire page.





## How to test?

- Check-out this branch.
- Run the project
- The pages should load in the browser

## Checklist
